src/openrct2/entity/Balloon.cpp

```diff
@@ -77,11 +84,12 @@ void Balloon::Press()
     }
 }
 
-void Balloon::Pop()
+void Balloon::Pop(bool silent)
 {
     popped = 1;
     frame = 0;
-    OpenRCT2::Audio::Play3D(OpenRCT2::Audio::SoundId::BalloonPop, { x, y, z });
+    if (!silent)
+        OpenRCT2::Audio::Play3D(OpenRCT2::Audio::SoundId::BalloonPop, { x, y, z });
```

Comment:
```suggestion
    if (!silent)
    {
        OpenRCT2::Audio::Play3D(OpenRCT2::Audio::SoundId::BalloonPop, GetLocation());
    }
```

Reply:
I've made the change. If braces are preferred it should be mentioned in the code style document